cmake/add_fctest.cmake

```diff
@@ -43,7 +43,7 @@ cmake_policy( SET CMP0064 NEW ) # Recognize ``TEST`` as operator for the ``if()`
 
   set( options           )
   set( single_value_args TARGET )
-  set( multi_value_args )
+  set( multi_value_args INCLUDEDIRS )
```

Comment:
To be consistent with ecbuild_add_test we should use `INCLUDES`, and they should already be part of the signature of add_fctest which just extends `ecbuild_add_test`.